tests/python/test_inverse_kinematics.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+import numpy as np
+from . import utils
+from gym_ignition.robots import sim
+import gym_ignition_models as models
+from gym_ignition.utils.inverse_kinematics_nlp import TargetType
+from gym_ignition.base.robot.robot_joints import JointControlMode, PID
+from gym_ignition.utils.inverse_kinematics_nlp import InverseKinematicsNLP
+
+# From: https://github.com/mkrizmancic/franka_gazebo/blob/master/config/default.yaml
+# These gains work well with a physics step of 1000 Hz.
+pid_gains_1000_hz = {
+    'panda_joint1': PID(p=50, i=0, d=20),
+    'panda_joint2': PID(p=10000, i=0, d=500),
+    'panda_joint3': PID(p=100, i=0, d=10),
+    'panda_joint4': PID(p=1000, i=0, d=50),
+    'panda_joint5': PID(p=100, i=0, d=10),
+    'panda_joint6': PID(p=100, i=0, d=10),
+    'panda_joint7': PID(p=10, i=0.5, d=0.1),
+    'panda_finger_joint1': PID(p=100, i=0, d=50),
+    'panda_finger_joint2': PID(p=100, i=0, d=50),
+}
```

Comment:
These gains are the same as the default ones.

Reply:
Yes, they are there on purpose since the default ones could change in the future. The test should be robust to that.